### ISSUE TITLE
Provide at least another theme

### ISSUE THREAD

**Issue by mariocasciaro** [hadronjs/hadron#8]

**Comment by rafinskipg** [comment]
Yeah, that would be great! or upload the documentation on how to add more themes.

**Comment by johnbrett** [comment]
@rafinskipg I've started work on one here, I'll try write some docs on it but as you can see the code is quite small. Any question feel free to get in touch. https://github.com/johnbrett/hadron-theme-shade

**Comment by rafinskipg** [comment]
@johnbrett Yep! My first main question is how to change the default image, :P It could be a great feature being able to change it via the admin interface, but this is related to hadron itself.
I'll take a look to your code, thanks!

**Comment by johnbrett** [comment]
Yeah I plan to add this when I get time, as well as an option to change the
profile image.

PRs welcome ;)

**Comment by rafinskipg** [comment]
Yeah, i have no time for that! Other way i would contribute.

**Comment by johnbrett** [comment]
@rafinskipg I've updated it now so you can change the image through the admin interface! Let me know if you try it out!

**Comment by almereyda** [comment]
You can also consider https://github.com/JacobPlaster/hadron-theme-wocka by @JacobPlaster

**Comment by mariocasciaro** [comment]
Yep, I should create a page with the list of currently available themes...
